packages/trader/src/Modules/Trading/Components/Elements/PayoutPerPoint/payout-per-point-mobile-input.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Icon, PageOverlay, Popover, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import WheelPickerMobile from '../../Form/WheelPicker/wheel-picker-mobile';
+import './payout-per-point.scss';
+import Fieldset from 'App/Components/Form/fieldset';
+import { LabelPairedChevronsDownCaptionRegularIcon, LabelPairedChevronsUpCaptionRegularIcon } from '@deriv/quill-icons';
+
+const PayoutPerPointMobileInput = ({
+    togglePayoutWheelPicker,
+    payoutChoices,
+    onPayoutClick,
+    selectedBarrier,
+    payout_per_point,
+    currency,
+}: {
+    togglePayoutWheelPicker: () => void;
+    payoutChoices: number[];
+    selectedBarrier: string;
+    onPayoutClick: (val: number) => void;
+    currency: string;
+    payout_per_point?: number;
+}) => {
+    const [initialPayout, setInitialPayout] = useState<number | null>(null);
+
+    const [payoutValue, setPayoutValue] = useState(payoutChoices[2] || payoutChoices[0]);
+
+    useEffect(() => {
+        if (initialPayout == null && payout_per_point) {
+            setInitialPayout(payout_per_point);
+        }
+    }, []);
+
+    const onSave = () => {
+        togglePayoutWheelPicker();
+        onPayoutClick(payoutValue);
+    };
+
+    const onClose = () => {
+        togglePayoutWheelPicker();
+        if (initialPayout) {
+            onPayoutClick(initialPayout);
+        }
+    };
+
+    const header_tooltip_text = (
+        <div className='trade-container__barriers-tooltip'>
+            <Localize i18n_default_text='You receive a payout at expiry if the spot price never touches or breaches the barrier throughout the contract duration. Otherwise, your contract will be terminated early.' />
+        </div>
+    );
+    return (
+        <PageOverlay onClickClose={onClose}>
+            <div className='payout-per-point-mobile'>
+                <Fieldset className='payout-per-point-mobile__header'>
+                    <div className='payout-per-point-mobile__header__title'>
+                        <Text size='xs' weight='bold' color='default' as='h1'>
+                            {localize('Payout per Point')}
```

Comment:
I'm not 100% sure, but I thought we try to use `<Localize/>` instead of `localize` were it's possible. Please, ignore if irrelevant)

Reply:
Sam in all other places